app/views/places/_form.erb

```diff
@@ -0,0 +1,35 @@
+<div class="row">
+  <div class="col-sm-12">
+    <%= form_for(@place, :html => { class: "form-horizontal", role: "form" }) do |f| %>
```

Comment:
old "hash rocket" notation, please use the new notation `html: { class: ...`

Reply:
done